### ISSUE TITLE
Tooltips fieldset on VAMC System Policies Page are not accessible

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#4049]
## ACs
- [ ] Fieldgroups on VAMC System Policies Pages, with tooltips, must be accessible. The current fieldsets with empty legends are not accessible. Nor are they really fieldsets, because they don't contain fields. 

## Implementation
- [ ] We should explore an approach that lets us add tooltips either to fieldsets, with legends, or to some other fieldgroup type (HTML element? A new custom fieldgroup type?), that doesn't require a legend for accessibility.

**Comment by kevwalsh** [comment]
i repurposed this issue to focus on accessibility of the tooltip sections.

**Comment by kaise-lafrai** [comment]
@kevwalsh @swirtSJW @rachel-kauff I've been doing some looking/tinkering around to find the right field group for our tooltip implementation but have found no working solution yet. HTML element field group type does not have a description field to add/edit the text in the tooltip.

Some options:
1. Add additional css styling to the `.centralized` class to target the legend > span and apply styling similar to what `.visually-hidden` does. Anywhere `centralized` class is used will then visually hide the fieldset legend span value.
2. Add additional logic to the `.add-tooltip` js code. We can target the legend span and add the `visually-hidden` class.

I want to run these next options by Steve:
3. Add a custom setting field to the fieldset widget that allows us to hide/show the label. Similar checkbox can be found on the HTML Element widget.
4. Add a description field in the HTML Element widget to add/edit the text for the tooltip.
5. Create an entirely new field group type that has all the settings we want for the tooltip.

Please let me know what you think.

**Comment by swirtSJW** [comment]
Summary of call Kaise and I just had:
The custom field group should have two text fields as settings:
1. Tooltip text:  If !empty, it outputs a tooltip.
2. Description text:  If !empty it outputs a the text at the top of the fieldgroup as visible html

If possible it should follow USWDS markup as much as possible.   
Should talk over tippy implementation with @acrollet

**Comment by rachel-kauff** [comment]
@kaise-lafrai @swirtSJW You're probably already aware, but @acrollet has been working on a tooltip for the admin toolbar this sprint [in this issue](https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3984). Not sure if there's anything he's doing that might help inform this.
